src/main/java/DilemmaDetector/ParameterizedPhilosophy.java

```diff
@@ -1,10 +1,15 @@
 package DilemmaDetector;
 
-public class ParameterizedPhilosophy {
+import DilemmaDetector.Consequences.ConsequenceType;
+
+import java.util.Map;
 
-    public static final int lifeValue = 10;
-    public static final int severeInjuryValue = 5;
-    public static final int lightlyInjuryValue = 1;
+public class ParameterizedPhilosophy {
+    public static final Map<ConsequenceType, Integer> healthValue = Map.of(
```

Comment:
Nie final, bo to użytkownik ustawi wagi + trochę inna koncepcja będzie, nie skalujemy tak samo obrażeń lekkich, ciężkich i śmierci tylko to użytkownik sobie ustawi. Także to jest do przerobienia, ale też musimy wiedzieć co chcemy.

Reply:
Takie w pełni niezależne kategorię sobie wyobrażałem do ustawień
           {"Human life", 1},
            {"Human severe injury", 0},
            {"Human lightly injury", 0},
            {"Animal life", 0},
            {"Animal severe injury", 0},
            {"Animal lightly injury", 0},
            {"Material damages per 1000$", 0},
            {"Breaking the law", 0}